segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -51,6 +53,9 @@
     public static final Property<Boolean> SECURE_ZK = Property.named("zk.connect.security.enable", false, "secureZK");
     public static final Property<String> ZK_TRUSTSTORE_LOCATION = Property.named("zk.connect.security.tls.trustStore.location", "", "zkTrustStore");
     public static final Property<String> ZK_TRUST_STORE_PASSWORD_PATH = Property.named("zk.connect.security.tls.trustStore.pwd.location", "", "zkTrustStorePasswordPath");
+    public static final Property<String> REST_LISTENING_HOST = Property.named("rest.listener.host", "localhost");
+    public static final Property<Integer> REST_LISTENING_PORT = Property.named("rest.listener.port", 6061);
+    public static final Property<Integer> HEALTH_CHECK_INTERVAL = Property.named("health.interval", 10);
```

Comment:
Always include the time unit in your property names. For example `health.interval.seconds` (I assume it's seconds, right?)